package.json

```diff
@@ -59,11 +60,12 @@
     "passport-google-oauth": "^0.2.0",
     "passport-local": "^1.0.0",
     "react": "^0.13.3",
+    "react-router" : "^1.0.0-beta3",
     "react-helmet": "^1.0.1",
-    "react-router": "1.0.0-beta3",
```

Comment:
@Faolain Would you mind changing this to rc1 and try fixing transitionTo errors? Or would you prefer the last beta merged first and then a separate PR for rc1?

I've tested it with rc1 apart from the the onEnter errors where it checks if the user is logged in, everything works well. Good work with the `match` change in server.jsx. Thanks a lot man!